### PR TITLE
python3.pkgs.tern: init at 2.5.0

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119995` at 7db5e44f run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.debut</li>
    <li>python38Packages.tern</li>
    <li>python39Packages.debut</li>
    <li>python39Packages.tern</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `fetchFromGitHub`.
Near pkgs/development/python-modules/tern/default.nix:4:3:
```
  |
4 | , fetchFromGitHub
  |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/tern/default.nix:51:0:
```
   |
51 |     description = "A software composition analysis tool and Python library that generates a Software Bill of Materials for container images and Dockerfiles.";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/debut/default.nix:28:0:
```
   |
28 |     description = "A python library to parse Debian deb822-style control and copyright files ";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/debut/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, chardet
+, attrs
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/debut/default.nix:7]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/debut/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  # configure phase, so remove it
+  preConfigure = ''
+    rm configure
+  '';
+
+  propagatedBuildInputs = [
+    chardet
+    attrs
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/debut/default.nix:26]: missing checkPhase and pythonImportsCheck.

File: pkgs/development/python-modules/tern/default.nix [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+, requests
+, stevedore
+, pbr
+, debut
+, regex
+, GitPython
+, prettytable
+, idna
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/tern/default.nix:16]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/tern/default.nix [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    dockerfile-parse
+    requests
+    stevedore
+    debut
+    regex
+    GitPython
+    prettytable
+    idna
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/tern/default.nix:48]: missing pythonImportsCheck.

File: pkgs/development/python-modules/debut/default.nix [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  meta = with lib; {
+    description = "A python library to parse Debian deb822-style control and copyright files ";
```
Comment (fabaff) [inline pkgs/development/python-modules/debut/default.nix:28]:
```suggestion
    description = "Python library to parse Debian deb822-style control and copyright files";
```

File: pkgs/development/python-modules/tern/default.nix [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  meta = with lib; {
+    description = "A software composition analysis tool and Python library that generates a Software Bill of Materials for container images and Dockerfiles.";
```
Comment (fabaff) [inline pkgs/development/python-modules/tern/default.nix:50]:
```suggestion
    description = "A software composition analysis tool and Python library that generates a Software Bill of Materials for container images and Dockerfiles";
```

see https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

**Pull request merged**